gson/src/main/java/com/google/gson/FieldAdapterFactory.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2008 Google Inc.
```

Comment:
Two small problems here. It’s not 2008, and it’s not Google. Please use “Copyright (C) 2017 Gson Authors”.